### ISSUE TITLE
Accessibility issue 2 - missing caption element for tables in Props

### ISSUE THREAD

**Issue by karypun** [UKHomeOffice/design-system#256]
**Pages affected:** all component pages
**Issue:** Missing <caption> element for tables within "Props" click to reveal. Found on every components page.
**Resolution:** Add in caption element

**Comment by daniel-ac-martin** [comment]
For future reference, this could be fixed by providing a caption prop here: https://github.com/daniel-ac-martin/NotGovUK/blob/master/lib/docs-components/src/Props.tsx#L41

In the GDS system they use the caption, "Primary options".
